data/campaigns/Winds_of_Fate/story/Morogor_Dialect.txt

```diff
@@ -0,0 +1,152 @@
+
+- Greatland -
+The Drakish word for "continent" (along with "great island").
+Usually used to refer to the Great Continent to the east of Morogor.
+
+- Hatchery -
+Where Drake eggs are cared for. Generally overseen and protected by members of the Clasher Caste. A library is often adjacent, where Clashers guard the Shells of Secrets.
+
+- Hatchling -
+A newly hatched Drake, about two thirds the size of an adult human but clumsy and largely defenseless.
+
+- Hunt -
+An organized, large scale hunt for Prey called by a high ranking Drake.
+The Drake that calls the Hunt is responsible for setting the end goal as well as the safety of the hunters.
+
+- Infestation, Infester -
+Any invasive creature that threatens Morogor's ecosystem and thereby the Drake's food supply.
+Usually designated as such by an Intendant of the Glider Caste.
+
+- Intendants -
+Advisors and lieutenants of a Dominant, usually one of each caste other than its own.
+
+- Intendant, First -
+Primary advisor to the Dominant.
+Tasked to take temporary command of the Flight if ever its Dominant is incapacitated.
+
+- Morogor -
+Also known as the Domain of Morogor
+Archipelago, located somewhere in the Great Ocean east of the Green Isle and west of the "Greatland" (Great Continent).
+
+- Prey -
+Any creature Drakes consider food, though generally the most dangerous ones, with "game" more often used to refer to easier Prey.
+
+- Shell of Secrets -
+A porcelain artifact on which the secret written history and technical knowledge of the Drakes is scribed.
+It is the shape and size of a drake egg and likewise guarded by the Clashers, the only Caste allowed to read its inscriptions.
+Its hollow interior allows it to float on a protective pool of water during the frequent earthquakes.
+
+- Spoken Memory -
+Also called the "Memory", "Glider Memory" and "Glider Spoken Memory".
+An oral history of the Drakes going back into the early times of Irdya, but generally known only to the Glider Caste.
+It is augmented by a vague sense of ancestral memory drakes possess.
+
+- Ways of Morogor -
+The truly enormous code of laws that govern Morogor and the highest authority in the land.
+They have accrued for nearly five thousand years and are more and more focused on managing territorial redistribution as Morogor loses ground to the surrounding ocean.
+
+- Winds of Fate -
+Also called the "Wind" and the "Winds".
+Glider belief that destiny is a kind of swirling complex of air currents, which they call Windpaths.
+These Windpaths cannot be seen, but can be felt "under your wings".
+Carefully choosing the right Windpath can deliver one to a desirable outcome, the wrong one leads to ruin.
+Gliders believe there is no escaping the Winds, you are either on one Windpath or another.
+
+- Windpath -
+See 'Winds of Fate'.
+
+- Wing -
+A small attack group of drakes.
+Generally a detachment from a much larger Flight, used in warfare or when a Hunt has been called.
```

Comment:
Perhaps there's no reason there should be a correlation, but I note most real world air forces (at least Anglophone ones) seem to use wing as a larger unit than flight.